.github/workflows/cd.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
```

Comment:
```suggestion
#      - master
```

Let's disable this, otherwise once MZ commits on master, the (then outdated) master will be published as continuous.

Reply:
We would need to modify the cd.yml in the master branch for that to work, because it looks to the workflows in whatever branch runs.

I figured it wouldn't be too much of a problem, since we'd be a few hours out from merging 0.9.15 back in once MZ makes the release. If one of the maintainers wants continuous to be completely removed from master on both branches, though, I'll rebase the PR to that (can a PR be applied to multiple branches?).

Reply:
Right, that's inconvenient :/

Either we deem those few hours acceptable, or we gotta change this on master as well (which might give a small merge conflict later)